### DISCUSSION TITLE
Preparation for Hacktoberfest

### DISCUSSION THREAD

**Discussion by EOEboh** [EddieHubCommunity/support#4984] [Ideas]
Hey community members, this discussion is being opened so we can effectively prepare and plan out for the upcoming month-long open source program called Hacktoberfest

**Comment by EOEboh** [comment]
> Hey community members, this discussion is being opened so we can effectively prepare and plan out for the upcoming month-long open source program called Hacktoberfest

If you wanna get an overview of how Hacktoberfest works, check out [how the Hacktoberfest 2021 guidelines were](https://blog.shahednasser.com/how-to-participate-in-hacktoberfest-2021/)

**Comment by eddiejaoude** [comment]
What ideas do you all have for `hacktoberfest`? Here are some ideas...

- Weekly live streams dedicated to `hacktoberfest` support
- Discord calls with screen sharing 
- Extra competitions + swag give aways
- EddieCon 2.0 where you all give talks about what you are working on
... share your ideas below 👇

**Reply by EOEboh** [reply]
I would love the discord calls and swag.
Eddie con is in a few days right?

**Reply by Njong392** [reply]
> What ideas do you all have for `hacktoberfest`? Here are some ideas...
> 
> * Weekly live streams dedicated to `hacktoberfest` support

Quick question. Are we going to have a hacktoberfest repo set up like the one the org already has? Or is the live stream going to be general tips on how to contribute to hacktoberfest etc etc

**Reply by EOEboh** [reply]
> > What ideas do you all have for `hacktoberfest`? Here are some ideas...
> > 
> > * Weekly live streams dedicated to `hacktoberfest` support
> 
> Quick question. Are we going to have a hacktoberfest repo set up like the one the org already has? Or is the live stream going to be general tips on how to contribute to hacktoberfest etc etc

Emy, for the first question, when Hacktoberfest starts, you are simply gonna search for repos on GitHub with the label tagged "Hacktoberfest".. anyone that suits you, you can contribute to.

As for the second question, let @eddiejaoude answer it

**Reply by eddiejaoude** [reply]
We haven't got EddieCon planned yet, I was thinking for October or a community video or both.

For the live streams, I am open to ideas, finding people issues for them to do could be interesting

**Reply by EOEboh** [reply]
> We haven't got EddieCon planned yet, I was thinking for October or a community video or both.
> 
> For the live streams, I am open to ideas, finding people issues for them to do could be interesting

Yeah that would make a lot of sense. Especially the good first issues

**Reply by Njong392** [reply]
> For the live streams, I am open to ideas, finding people issues for them to do could be interesting

Oh okay. I see. I thought we were going to set up another repo for hacktoberfest like the one we already have. Helping people find issues and pull requests during the live streams sounds great too. That way, it can be both an info session and an opportunity for people to contribute :+1:

**Reply by schmelto** [reply]
Before Hacktoberfest we should go through our issues and add to selected ones the `hacktoberfest` label. Also we should add the hacktoberfest label to the repos.

**Reply by EOEboh** [reply]
@schmelto great suggestion.
Especially for those new to open source, we could make sure to source out as many good first issues as possible

**Reply by Aditya-150** [reply]
> We haven't got EddieCon planned yet, I was thinking for October or a community video or both.
> 
> For the live streams, I am open to ideas, finding people issues for them to do could be interesting

Yeah! I agree . I think a session on git and GitHub basic will also be helpful.
